packages/adapter-node/README.md

```diff
@@ -40,11 +40,44 @@ Enables precompressing using gzip and brotli for assets and prerendered pages. I
 By default, the server will accept connections on `0.0.0.0` using port 3000. These can be customised with the `PORT` and `HOST` environment variables:
 
 ```
-HOST=127.0.0.1 PORT=4000 node build
+HOST=127.0.0.1 PORT=4000 node build/server
```

Comment:
I would expect the default server to be exposed as `index` still so that this line wouldn't change

Reply:
To quote @Rich-Harris here:
> What if instead of `asRequestHandler`, the adapter outputted two files — a `server.js` (similar to the current `index.js`) but also an `index.js` that exposed the bits you need, and which `server.js` imported from?
> 
> That way you could either run the 'reference implementation' server-in-a-box — `node build/server` — or you could import the handler and do whatever you want with it.

Reply:
Doing that would be a backwards incompatible change. That'd be okay since we're in beta if there's some benefit to it, but it doesn't seem like there's any real benefit here that would justify breaking the deployment process of everyone who already has a project running with `adapter-node`

Reply:
I switched the filenames. `index.js` ist now `handler.js` again,
and `server.js` is now `index.js`.
So in the end, using the final adapter-node you can call `node build` and get the reference server implementation.
To use the handler/middleware, you have to import `build/handler.js`into you own server implementation.